vissl/data/ssl_transforms/tensor_to_img_pil.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+from io import BytesIO
+
+from classy_vision.dataset.transforms import register_transform
+from classy_vision.dataset.transforms.classy_transform import ClassyTransform
+from PIL import Image
+
+
+@register_transform("TensorToImgPil")
+class TensorToImgPil(ClassyTransform):
```

Comment:
Yeah -- not really. It should probably be moved to internal.

I refrained from doing this because it would then require some refactoring of the code here: https://github.com/facebookresearch/vissl/pull/317/files#diff-82aa56b0a134fa104a6d729561ff3aaf76e2b69c735037d7c8858480dafc5622R140 to be because this class would no longer be importable.

Reply:
I move to internal in subsequent PR.